src/RoboFile.php

```diff
@@ -290,13 +296,13 @@ public function dbDumpDataPartial(
     }
 
     /**
-     * Dump GRANTs for appropriate users to a file
+     * Dump CREATE GRANTS for users defined in getGrantUsers to a file
      */
     public function dbDumpGrants(
         array $opts = [
             'file'       => InputOption::VALUE_REQUIRED,
             'password'   => InputOption::VALUE_REQUIRED,
-            'appendFile' => false
+            'append'     => false
```

Comment:
This task already had append functionality since it uses mysql CLI, but I changed this parameter so that the `--append` flag is consistent throughout all tasks. As this is a BC breaking change, I'll make a major release.